client/packages/system/src/Encounter/DetailView/Toolbar.tsx

```diff
@@ -19,12 +19,17 @@ interface ToolbarProps {
   onChange: (patch: Partial<EncounterFragment>) => void;
 }
 export const Toolbar: FC<ToolbarProps> = ({ onChange }) => {
-  const { data: encounter } = useEncounter.document.get();
+  const { mutate: fetchEncounter, data: encounter } =
```

Comment:
this was updated to use a mutation in order to fix an infinite render issue.. and I forgot to update the toolbar too.